trailblazer/apps/slurm/api.py

```diff
@@ -1,10 +1,50 @@
+import subprocess
+from pathlib import Path
 from typing import Callable, Dict, List, Optional
 
 from trailblazer.apps.slurm.models import SqueueResult
 from trailblazer.apps.slurm.utils import formatters
 from trailblazer.constants import FileFormat, SlurmJobStatus, TrailblazerStatus
 from trailblazer.exc import EmptySqueueError
-from trailblazer.io.controller import ReadStream
+from trailblazer.io.controller import ReadFile, ReadStream
+
+
+def _get_squeue_jobs_flag_input(slurm_job_id_file_content: Dict[str, List[str]]) -> str:
+    """Return a string of comma separated SLURM job ids to be used as input for squeue jobs flag."""
+    job_ids: List[str] = []
+    for slurm_job_ids in slurm_job_id_file_content.values():
+        [job_ids.append(str(job_id)) for job_id in slurm_job_ids]
```

Comment:
```suggestion
    slurm_job_ids: List[List[str]] = slurm_job_id_file_content.values()
    job_ids: List[str] = [job_id for job_id in job_ids for job_ids in slurm_job_ids]
```

Reply:
Not a fan of nested for loops within a list comprehension. I think it is hard to understand. I would like to keep the current implementation. Ok?